src/bodyScrollLock.js

```diff
@@ -141,8 +132,8 @@ const restorePositionSetting = () => {
 
     // Restore styles
     document.body.style.position = previousBodyPosition.position;
-    document.body.style.top = previousBodyPosition.top;
-    document.body.style.left = previousBodyPosition.left;
+    document.body.style.top = `${previousBodyPosition.top}px`;
```

Comment:
These props contain 'px' already. There is no need to edit this lines

Reply:
Thanks for the review, I removed the extra "px".